C2GUILauncher/src/ViewModels/LauncherViewModel.cs

```diff
@@ -119,6 +122,25 @@ public class LauncherViewModel {
             }
         }
 
+        private IEnumerable<string> GetINIFileArgs() {
+            if(Settings.ExtraINIDir.Trim() == "" || !Directory.Exists(Settings.ExtraINIDir)) 
+                return new List<string>() { };
+
+            var iniFiles = Directory.EnumerateFiles(Settings.ExtraINIDir, "*.ini");
+
+            var iniArgs =
+                iniFiles.Select(iniFilePath => {
+                    var absoluteiniFilePath = Path.GetFullPath(iniFilePath);
+
+                    var iniFileName = Path.GetFileName(iniFilePath);
+                    var cliFlagName = iniFileName.ToUpper().Replace("DEFAULT", "DEF").Replace(".INI", "INI");
```

Comment:
Sets the flag name based on https://docs.unrealengine.com/4.27/en-US/ProductionPipelines/CommandLineArguments/

Effectively, DefaultFoo.ini becomes DEFFOOINI, Foo.ini becomes FOOINI